README.md

```diff
@@ -39,18 +68,38 @@ Tests will run inside a Docker container and run a dedicated docker daemon as do
 
 AWS specific features (VPC integration, metadata service proxy, GPU, EFS, ...) are disabled during these tests.
 
-## Generating Go code based on the agent protobuf definition
+## Generated Code 
+There are places inside of the executor where we've checked in binaries, or prebuilt pieces of code. This may be considered harmful by some, but from the perspective of pragmatism, we have such code in:
+
+In order to generate code you need `go-bindata`, which you can get via:
 
 ```sh-session
-make protogen
+go get -u github.com/jteeuwen/go-bindata/...
 ```
 
-### Force the protobuf definition to be re-fetched and Go code re-generated
+### `vpc/bpf`
+There are two pieces of generated code:
+
+* `vpc/bpf/filter.o`: This is compiled from `vpc/bpf/filter.c`. We ship this precompiled because it changes infrequently, and it requires LLVM, and the full kernel headers
+* `vpc/bpf/filter/filter.go`: This is a go package generated using go-bindata based upon `vpc/bpf/filter.o`
+
+In order to regenerate this data, if you edit `vpc/bpf/filter.c`, you can do so by running:
 
 ```sh-session
-make clean-proto-defs && make protogen
+make -C vpc/bpf/
 ```
 
+### api/netflix/titus
+If you update the dependency `github.com/Netflix/titus-api-definitions/src/main/proto/netflix/titus`, you will need to regenerate the Go proto definitions. 
+
+You must first install the protobuf toolchain, and the Go protobuf compiler. More documentation on that can be found [in the protobuf repo](https://github.com/golang/protobuf/blob/master/README.md#installation).
```

Comment:
different versions of the `protoc` compiler will generate different code, and some can potentially fail. This was the main reason the `protoc` stuff was containerized and pinned to a particular version.

Reply:
Yeah, since the generated code is checked in (by us). We should be able to verify it.